packages/data/src/index.ts

```diff
@@ -110,11 +112,11 @@ export const combineReducers =
  *
  * @return {Object} Object containing the store's promise-wrapped selectors.
```

Comment:
It's better to remove this return type in JSDoc to have TS dictate the type.
```suggestion
 * @return Object containing the store's promise-wrapped selectors.
```